src/models/adr-details.ts

```diff
@@ -9,24 +10,101 @@ export interface AdrDetails {
   brakeDeclarationsSeen?: boolean;
   brakeDeclarationIssuer?: string;
   brakeEndurance?: boolean;
-  weight?: string;
-  compatibilityGroupJ?: boolean;
+  weight?: number;
+  newCertificateRequested?: boolean;
+  compatibilityGroupJ?: compatibilityGroupJEnum;
   documents?: string[];
-  permittedDangerousGoods?: string[];
-  additionalExaminerNotes?: string;
+  permittedDangerousGoods?: permittedDangerousGoodsEnum[];
+  additionalExaminerNotes?: AdditionalExaminerNotes;
   applicantDetails?: ApplicantDetails;
-  memosApply?: string[];
+  dangerousGoods?: boolean;
```

Comment:
This is not required as it is a front end field for UX

Reply:
Same as `newCertificateRequested`